--- a/cyrus-imapd.spec	2020-04-14 16:20:21.006799690 +0200
+++ b/cyrus-imapd.spec	2020-04-14 16:37:52.449966296 +0200
@@ -65,6 +65,9 @@
 ##
 Patch0003:          0003-Canonification-for-multiple-domains.patch
 
+# see https://lists.andrew.cmu.edu/pipermail/cyrus-devel/2016-November/003919.html
+Patch11:             cyrus_filter_kolab_mailboxes.patch
+
 BuildRoot:          %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 
 ##
@@ -259,6 +262,8 @@
 %prep
 %setup -q -n %{name}-%{tag_version}-%{revision}-%{git_hash}
 
+%patch11 -p1
+
 %if 0%{?with_bdb} < 1
 sed -i -e 's/,berkeley//g' cunit/aaa-db.testc
 sed -r -i -e 's/"berkeley(|-[a-z-]+)", //g' lib/imapoptions
