--- a/kolab.spec	2019-03-08 22:13:54.096253181 +0100
+++ b/kolab.spec	2019-03-08 22:14:25.444140522 +0100
@@ -59,11 +59,6 @@
 Summary:        Kolab Groupware IMAP Component
 Group:          Applications/System
 Requires:       cyrus-imapd
-%if %{?_arch} == "ppc64le" && 0%{?fedora}
-# Do not require Guam on fedora/ppc64le
-%else
-Requires:       guam
-%endif
 Requires:       kolab-saslauthd
 Requires:       pykolab
 
@@ -131,18 +126,6 @@
 %package mta
 Summary:        The Kolab Groupware Mail Transfer Agent (MTA) meta-package
 Group:          Applications/System
-Requires:       amavisd-new
-
-%if 0%{?rhel} > 6 || 0%{?fedora} > 0
-Requires:       clamav-update
-%endif
-
-%if 0%{?with_systemd}
-Requires:       clamav-server-systemd
-%else
-Requires:       clamav-server-sysvinit
-%endif
-Requires:       clamav-update
 
 Requires:       postfix
 Requires:       postfix-kolab
@@ -197,7 +180,6 @@
 
 Requires:       roundcubemail-plugin-contextmenu
 Requires:       roundcubemail-plugins-kolab
-Requires:       roundcubemail-skin-kolab
 
 %description webclient
 This is the Kolab Groupware web client meta-package
