--- a/pykolab.spec	2017-04-08 15:31:40.370572889 +0200
+++ b/pykolab.spec	2017-04-08 15:31:40.494572001 +0200
@@ -40,6 +40,19 @@
 
 Patch0001:          pykolab-0.8-patch-out-manticore.patch
 
+# from initTBitsISP.sh
+Patch8:             pykolab_do_not_rename_existing_mailbox_T3315.patch
+Patch9:             lastLoginTBitsAttribute-pykolab.patch
+Patch10:            allowPrimaryEmailAddressFromDomain.patch
+Patch11:            canonification_via_uid_pykolab.patch
+Patch12:            logLoginData.patch
+Patch13:            disableSpamFilter.patch
+Patch14:            disableSpamFilter2.patch
+Patch15:            fixPykolabIMAPKeepAlive.patch
+Patch16:            onlyAllowKolabUsersToAuthViaSasl.patch
+Patch17:            pykolab_wap_client_unverified_context_localhost.patch
+Patch18:            kolab_lam_invalid_mailbox_name.patch
+
 BuildRoot:          %{_tmppath}/%{name}-%{version}-%{release}-root
 BuildArch:          noarch
 
@@ -245,6 +252,18 @@
 %patch0001 -p1
 %endif
 
+%patch8 -p1
+%patch9 -p1
+%patch10 -p1
+%patch11 -p1
+%patch12 -p1
+%patch13 -p1
+%patch14 -p1
+%patch15 -p1
+%patch16 -p1
+%patch17 -p1
+%patch18 -p1
+
 %build
 autoreconf -v || automake --add-missing && autoreconf -v
 %configure
