--- a/roundcubemail-plugins-kolab.spec	2020-04-14 16:24:35.189719091 +0200
+++ b/roundcubemail-plugins-kolab.spec	2020-04-14 16:28:54.408576328 +0200
@@ -43,7 +43,7 @@
 Name:           roundcubemail-plugins-kolab
 Version:        3.5.4
 
-Release:        2.4%{?dist}.kolab_16
+Release:        103.tbits%(date +%%Y%%m%%d)%{?dist}
 
 Summary:        Kolab Groupware plugins for Roundcube Webmail
 
@@ -63,6 +63,12 @@
 
 Patch0000:      roundcubemail-plugins-kolab-3.4-kolab-files-manticore-api.patch
 
+# from initSetupKolabPatches.sh
+Patch10:         roundcubeStorageMariadbBug4883.patch
+Patch11:         roundcube_calendar_etc_timezone_T2666.patch
+
+Patch12:         canonification_via_uid_roundcube.patch
+
 BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 BuildArch:      noarch
 
@@ -1407,6 +1413,10 @@
 
 %patch0000 -p1
 
+%patch10 -p1
+%patch11 -p1
+%patch12 -p1
+
 find -type d -name "helpdocs" -exec rm -rvf {} \; 2>/dev/null || :
 
 rm -rf plugins/kolab_zpush
