--- a/roundcubemail.spec	2020-04-14 16:30:24.882787481 +0200
+++ b/roundcubemail.spec	2020-04-14 16:35:33.136708218 +0200
@@ -56,7 +56,7 @@
 Name:           roundcubemail
 Version:        1.4.3.6
 
-Release:        1.3%{?dist}.kolab_16
+Release:        103.tbits%(date +%%Y%%m%%d)%{?dist}
 
 Summary:        Round Cube Webmail is a browser-based multilingual IMAP client
 
@@ -79,6 +79,10 @@
 Patch201:       default-configuration.patch
 Patch202:       roundcubemail-1.4-beta86-plugin-enigma-homedir.patch
 
+Patch400:       optional_disable_addressbook_export.patch
+Patch401:       roundcubemail_messages.patch
+Patch402:       backport_managesieve_forwards.patch
+
 BuildArch:      noarch
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root%(%{__id_u} -n)
 
@@ -1537,6 +1541,13 @@
 %patch202 -p1
 %endif
 
+%patch400 -p1
+%patch401 -p1
+%patch402 -p1
+
+# do not use timezones like Etc/GMT-1 in the jstz library (see also https://bitbucket.org/pellepim/jstimezonedetect/issues/168/ignore-timezones-like-etc-gmt-1)
+sed -i 's#"UTC"===a)#"UTC"===a)\&\&a.indexOf("Etc")<0#' public_html/program/js/jstz.min.js
+
 # Remove the results of patching when there's an incidental offset
 find . -type f -name "*.orig" | while read file; do
     rm -rvf ${file}
@@ -2394,7 +2405,7 @@
 
 %check
 pushd %{name}-%{version}%{?dash_rel_suffix}/tests
-phpunit --debug || :
+#phpunit --debug || :
 popd
 
 %clean
