--- a/roundcubemail.orig	2018-11-30 20:57:25.742661536 +0100
+++ b/roundcubemail.spec	2018-11-30 21:00:55.789859547 +0100
@@ -47,7 +47,7 @@
 %global tmpdir /var/lib/roundcubemail
 
 Name:           roundcubemail
-Version:        1.3.6
+Version:        1.3.8
 
 Release:        103.tbits%(date +%%Y%%m%%d)%{?dist}
 
@@ -69,13 +69,10 @@
 
 Source200:      2017111400.sql
 
-Patch0002:      0002-Parse-all-quotas-from-GETQUOTAROOT-6280.patch
-Patch0003:      0003-Update-changelog.patch
-Patch0004:      0004-Fix-bug-where-some-escape-sequences-in-html-styles-c.patch
-Patch0005:      0005-Fix-bug-where-some-forbidden-characters-on-Cyrus-IMA.patch
-
 Patch201:       default-configuration.patch
 Patch202:       roundcubemail-1.3.6-plugin-enigma-homedir.patch
+Patch400:       optional_disable_addressbook_export.patch
+Patch401:       roundcubemail_messages.patch
 
 BuildArch:      noarch
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root%(%{__id_u} -n)
@@ -106,7 +103,7 @@
 BuildRequires:  php-pear(MDB2_Driver_mysqli)
 BuildRequires:  php-pear(Net_IDNA2)
 BuildRequires:  php-pear(Net_LDAP2)
-BuildRequires:  php-pear(Net_LDAP3)
+BuildRequires:  php-composer(kolab/Net_LDAP3)
 BuildRequires:  php-pear(Net_Sieve)
 BuildRequires:  php-pear(Net_SMTP)
 BuildRequires:  php-pear(Net_Socket)
@@ -188,7 +185,7 @@
 
 %if 0%{?plesk} < 1
 Requires:       php-pear(Net_LDAP2)
-Requires:       php-pear(Net_LDAP3)
+Requires:       php-composer(kolab/Net_LDAP3)
 %endif
 
 Requires:       %{name}(core-assets) = %{?epoch:%{epoch}:}%{version}-%{release}
@@ -1075,13 +1072,13 @@
 cp -vf %{SOURCE102} plugins/password/config.inc.php.dist
 %endif
 
-%patch0002 -p1
-%patch0003 -p1
-%patch0004 -p1
-%patch0005 -p1
-
 %patch201 -p1
 %patch202 -p1
+%patch400 -p1
+%patch401 -p1
+
+# do not use timezones like Etc/GMT-1 in the jstz library (see also https://bitbucket.org/pellepim/jstimezonedetect/issues/168/ignore-timezones-like-etc-gmt-1)
+sed -i 's#"UTC"===a)#"UTC"===a)\&\&a.indexOf("Etc")<0#' public_html/program/js/jstz.min.js
 
 # Remove the results of patching when there's an incidental offset
 find . -type f -name "*.orig" -delete
@@ -1846,7 +1843,7 @@
 
 %check
 pushd %{name}-%{version}/tests
-phpunit --debug || :
+#phpunit --debug || :
 popd
 
 %clean
