--- a/roundcubemail-plugins-kolab.spec	2017-06-14 09:02:14.709793334 +0200
+++ b/roundcubemail-plugins-kolab.spec	2017-06-14 10:30:24.683577971 +0200
@@ -31,7 +31,7 @@
 Name:           roundcubemail-plugins-kolab
 Version:        3.4

-Release:        46%{?dot_rel_suffix}%{?dist}
+Release:        100.tbits%(date +%%Y%%m%%d)%{?dist}

 Summary:        Kolab Groupware plugins for Roundcube Webmail
 
@@ -46,8 +46,17 @@
 Source102:      plesk.kolab_folders.inc.php
 Source103:      plesk.libkolab.inc.php
 
+Source200:      ude_login.inc.php
+Source201:      ude_login.php
+
 Patch1001:      roundcubemail-plugins-kolab-3.3-kolab-files-manticore-api.patch
 
+# from initSetupKolabPatches.sh
+Patch10:         roundcubeStorageMariadbBug4883.patch
+Patch11:         roundcube_calendar_etc_timezone_T2666.patch
+
+Patch12:         canonification_via_uid_roundcube.patch
+
 BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 BuildArch:      noarch
 
@@ -85,6 +93,7 @@
 Requires:       roundcubemail(plugin-odfviewer) = %{?epoch:%{epoch}:}%{version}-%{release}
 Requires:       roundcubemail(plugin-pdfviewer) = %{?epoch:%{epoch}:}%{version}-%{release}
 Requires:       roundcubemail(plugin-tasklist) = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:       roundcubemail(plugin-ude_login) = %{?epoch:%{epoch}:}%{version}-%{release}
 Obsoletes:      roundcubemail-kolab < %{version}-%{release}
 Provides:       roundcubemail-kolab = %{version}-%{release}
 
@@ -135,6 +144,17 @@
 %description -n roundcubemail-plugin-kolab_2fa
 Plugin kolab_2fa
 
+%package -n roundcubemail-plugin-ude_login
+Summary:        Plugin ude_login
+Group:          Applications/Internet
+Requires:       roundcubemail(core) >= %{roundcube_version}
+Requires:       roundcubemail(plugin-ude_login-assets) = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:       roundcubemail(plugin-ude_login-skin) = %{?epoch:%{epoch}:}%{version}-%{release}
+Provides:       roundcubemail(plugin-ude_login) = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description -n roundcubemail-plugin-ude_login
+Plugin ude_login
+
 %package -n roundcubemail-plugin-kolab_activesync
 Summary:        Plugin kolab_activesync
 Group:          Applications/Internet
@@ -387,6 +418,14 @@
 %description -n roundcubemail-plugin-kolab_2fa-assets
 Plugin kolab_2fa Assets
 
+%package -n roundcubemail-plugin-ude_login-assets
+Summary:        Plugin ude_login Assets
+Group:          Applications/Internet
+Provides:       roundcubemail(plugin-ude_login-assets) = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description -n roundcubemail-plugin-ude_login-assets
+Plugin ude_login Assets
+
 %package -n roundcubemail-plugin-kolab_activesync-assets
 Summary:        Plugin kolab_activesync Assets
 Group:          Applications/Internet
@@ -591,6 +638,18 @@
 %description -n roundcubemail-plugin-kolab_2fa-skin-larry
 Plugin kolab_2fa / Skin larry
 
+%package -n roundcubemail-plugin-ude_login-skin-larry
+Summary:        Plugin ude_login / Skin larry
+Group:          Applications/Internet
+Requires:       roundcubemail(plugin-ude_login) = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:       roundcubemail(skin-larry) >= %{roundcube_version}
+Requires:       roundcubemail(plugin-ude_login-skin-larry-assets) = %{?epoch:%{epoch}:}%{version}-%{release}
+Provides:       roundcubemail(plugin-ude_login-skin) = %{?epoch:%{epoch}:}%{version}-%{release}
+Provides:       roundcubemail(plugin-ude_login-skin-larry) = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description -n roundcubemail-plugin-ude_login-skin-larry
+Plugin ude_login / Skin larry
+
 %package -n roundcubemail-plugin-kolab_activesync-skin-larry
 Summary:        Plugin kolab_activesync / Skin larry
 Group:          Applications/Internet
@@ -735,6 +806,14 @@
 %description -n roundcubemail-plugin-kolab_2fa-skin-larry-assets
 Plugin kolab_2fa / Skin larry (Assets Package)
 
+%package -n roundcubemail-plugin-ude_login-skin-larry-assets
+Summary:        Plugin ude_login / Skin larry (Assets)
+Group:          Applications/Internet
+Provides:       roundcubemail(plugin-ude_login-skin-larry-assets) = %{?epoch:%{epoch}:}%{version}-%{release}
+
+%description -n roundcubemail-plugin-ude_login-skin-larry-assets
+Plugin ude_login / Skin larry (Assets Package)
+
 %package -n roundcubemail-plugin-kolab_activesync-skin-larry-assets
 Summary:        Plugin kolab_activesync / Skin larry (Assets)
 Group:          Applications/Internet
@@ -856,6 +944,13 @@
 
 %patch0001 -p1
 %patch1001 -p1
+%patch10 -p1
+%patch11 -p1
+%patch12 -p1
+ 
+mkdir -p plugins/ude_login
+cp -af %{SOURCE200} plugins/ude_login/config.inc.php.dist
+cp -af %{SOURCE201} plugins/ude_login/ude_login.php
 
 find -type d -name "helpdocs" -exec rm -rvf {} \; 2>/dev/null || :
 
@@ -1281,6 +1386,11 @@
     %{__rm} -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted"
 fi
 
+%pre -n roundcubemail-plugin-ude_login
+if [ -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted" ]; then
+    %{__rm} -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted"
+fi
+
 %pre -n roundcubemail-plugin-kolab_activesync
 if [ -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted" ]; then
     %{__rm} -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted"
@@ -1426,6 +1541,21 @@
     touch %{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted
 fi
 
+%posttrans -n roundcubemail-plugin-ude_login
+if [ ! -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted" ]; then
+    if [ -f "%{php_inidir}/apc.ini" -o -f "%{php_inidir}/apcu.ini" ]; then
+        if [ ! -z "$(grep ^apc.enabled=1 %{php_inidir}/apc{,u}.ini)" ]; then
+%if 0%{?with_systemd}
+            /bin/systemctl condrestart %{httpd_name}.service
+%else
+            /sbin/service %{httpd_name} condrestart
+%endif
+        fi
+    fi
+    %{__mkdir_p} %{_localstatedir}/lib/rpm-state/roundcubemail/
+    touch %{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted
+fi
+
 %posttrans -n roundcubemail-plugin-kolab_activesync
 if [ ! -f "%{_localstatedir}/lib/rpm-state/roundcubemail/httpd.restarted" ]; then
     if [ -f "%{php_inidir}/apc.ini" -o -f "%{php_inidir}/apcu.ini" ]; then
@@ -1775,6 +1920,10 @@
 %defattr(-,root,root,-)
 %attr(0640,root,%{httpd_group}) %config(noreplace) %{confdir}/kolab_2fa.inc.php
 
+%files -n roundcubemail-plugin-ude_login -f plugin-ude_login.files
+%defattr(-,root,root,-)
+%attr(0640,root,%{httpd_group}) %config(noreplace) %{confdir}/ude_login.inc.php
+
 %files -n roundcubemail-plugin-kolab_activesync -f plugin-kolab_activesync.files
 %defattr(-,root,root,-)
 %attr(0640,root,%{httpd_group}) %config(noreplace) %{confdir}/kolab_activesync.inc.php
@@ -1855,6 +2008,9 @@
 %files -n roundcubemail-plugin-kolab_2fa-assets -f plugin-kolab_2fa-assets.files
 %defattr(-,root,root,-)
 
+%files -n roundcubemail-plugin-ude_login-assets -f plugin-ude_login-assets.files
+%defattr(-,root,root,-)
+
 %files -n roundcubemail-plugin-kolab_activesync-assets -f plugin-kolab_activesync-assets.files
 %defattr(-,root,root,-)
 
@@ -1924,6 +2083,9 @@
 %files -n roundcubemail-plugin-kolab_2fa-skin-larry -f plugin-kolab_2fa-skin-larry.files
 %defattr(-,root,root,-)
 
+%files -n roundcubemail-plugin-ude_login-skin-larry -f plugin-ude_login-skin-larry.files
+%defattr(-,root,root,-)
+
 %files -n roundcubemail-plugin-kolab_activesync-skin-larry -f plugin-kolab_activesync-skin-larry.files
 %defattr(-,root,root,-)
 
@@ -1963,6 +2128,9 @@
 %files -n roundcubemail-plugin-kolab_2fa-skin-larry-assets -f plugin-kolab_2fa-skin-larry-assets.files
 %defattr(-,root,root,-)
 
+%files -n roundcubemail-plugin-ude_login-skin-larry-assets -f plugin-ude_login-skin-larry-assets.files
+%defattr(-,root,root,-)
+
 %files -n roundcubemail-plugin-kolab_activesync-skin-larry-assets -f plugin-kolab_activesync-skin-larry-assets.files
 %defattr(-,root,root,-)
 
