--- a/lib/locale/de_DE.php	2015-03-16 15:24:39.239710901 +0100
+++ b/lib/locale/de_DE.php	2015-03-16 15:25:07.027710866 +0100
@@ -3,9 +3,9 @@
 $LANG['about.community'] = 'Dies ist die Community-Edition des <b>Kolab Servers</b>.';
 $LANG['about.kolab'] = 'Kolab';
 $LANG['about.kolabsys'] = 'Kolab Systems';
-$LANG['about.support'] = 'Professionelle Unterstützung ist erhältlich bei <a href="http://kolabsys.com">Kolab Systems</a>.';
+$LANG['about.support'] = 'Wir helfen Ihnen gerne!';
 $LANG['about.technology'] = 'Technologie';
-$LANG['about.warranty'] = 'Sie kommt mit <b>keinen Garantien</b> und wird normalerweise ohne professionelle Unterstützung eingesetzt. Hilfe und weitere Informationen finden Sie auf der <a href="http://kolab.org">Web-Seite</a> und im <a href="http://wiki.kolab.org">Wiki</a>.';
+$LANG['about.warranty'] = 'Diese Version wird von <a href="http://www.tbits.net">TBits.net</a> bereitgestellt und gepflegt.';
 
 $LANG['aci.new'] = 'Neu...';
 $LANG['aci.edit'] = 'Bearbeiten...';
--- a/lib/locale/en_US.php	2015-03-16 15:25:47.393710942 +0100
+++ b/lib/locale/en_US.php	2015-03-16 15:27:02.043710880 +0100
@@ -3,9 +3,9 @@
 $LANG['about.community'] = 'This is the Community Edition of the <b>Kolab Server</b>.';
 $LANG['about.kolab'] = 'Kolab';
 $LANG['about.kolabsys'] = 'Kolab Systems';
-$LANG['about.support'] = 'Professional support is available from <a href="http://kolabsys.com">Kolab Systems</a>.';
+$LANG['about.support'] = 'We are happy to help you!';
 $LANG['about.technology'] = 'Technology';
-$LANG['about.warranty'] = 'It comes with absolutely <b>no warranties</b> and is typically run entirely self supported. You can find help & information on the community <a href="http://kolab.org">web site</a> & <a href="http://wiki.kolab.org">wiki</a>.';
+$LANG['about.warranty'] = 'This edition has been installed and is being maintained by <a href="http://www.tbits.net">TBits.net</a>';
 
 $LANG['aci.new'] = 'New...';
 $LANG['aci.edit'] = 'Edit...';

